headless/headless-commerce/headless-commerce-admin-inventory-impl/src/main/java/com/liferay/headless/commerce/admin/inventory/internal/resource/v1_0/WarehouseItemResourceImpl.java

```diff
@@ -167,8 +167,7 @@ public WarehouseItem getWarehouseItemByExternalReferenceCode(
 		}
 
 		if ((start == null) && (end == null)) {
-			throw new CommerceInventoryInvalidDateException(
-				"At least one date should be defined");
+			start = new Date();
```

Comment:
@marco-leo not 100% about this change, but I think it make sense.. let me know what do you think. Thank you.